flask_project/campaign_manager/sqlalchemy_session.py

```diff
@@ -0,0 +1,21 @@
+import os
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+
+if 'DATABASE_URL' in os.environ:
```

Comment:
this if/elif/else doesnt look good. Tests might run on DATABASE_URL.

Reply:
This should maybe rather be tested for in global settings?

Reply:
@cchristelis yes, it is better to test this global settings (config) file, I have changed it accordingly and will update it with a commit soon.